### PR TITLE
Navigation drawer implementation

### PR DESCRIPTION
**Pull request by Meisolsson**
From the ideas and mocks in #664 I created this 
screenshot_2015-01-12-13-15-07 1

If there are any ideas or problems I'll look at it

### CODE REVIEW

File: app/src/main/java/com/github/mobile/ui/DialogFragmentHelper.java [hunk 1]
```diff
@@ -54,8 +56,8 @@
      * @param arguments
      * @param tag
      */
-    protected static void show(DialogFragmentActivity activity,
-            DialogFragmentHelper fragment, Bundle arguments, String tag) {
+    protected static void show(FragmentActivity activity,
```
Comment (fadils) [inline app/src/main/java/com/github/mobile/ui/DialogFragmentHelper.java]: Perhaps, I miss something. Anything wrong with RoboActionBarActivity?

**Comment by Meisolsson** [comment]
Well since none of the classes that uses that use RoboActionBarActivity this is a better way of doing it

**Comment by fadils** [comment]
@jonanp do you have something to add to this PR? Otherwise, should squash the commits and merge.

**Comment by fadils** [comment]
@Meisolsson could you please rebase on the latest PR you made so that we can merge this? It all looks good to me.

**Comment by fadils** [comment]
@Meisolsson Awesome work. Thanks!

**Comment by Meisolsson** [comment]
Still problems with the merge, we need to remove <code>HomeActivity</code> since it's not in use and alot of old things that it used were deleted in my PR. So for the project to compile we need to remove it.

**Pull request merged**